packages/themes/src/presets/kepler.css

```diff
@@ -167,5 +167,5 @@
     rgba(120, 119, 198, 0.3),
     transparent
   );
-  height: 100%;
+  height: 100vh;
```

Comment:
This is probably a breaking change. 100vh is not always the height of references.

Reply:
This just controls the size of a gradient, not a breaking change